public/components/common/flyouts/wz-flyout.tsx

```diff
@@ -10,21 +10,11 @@
  * Find more information about this on the LICENSE file.
  */
 
-import React, { Component, Fragment } from 'react';
-import { EuiFlyout, EuiOverlayMask, EuiOutsideClickDetector } from '@elastic/eui';
+import React from 'react';
+import { EuiFlyout } from '@elastic/eui';
 
-export const WzFlyout = ({children, flyoutProps = {}, overlayMaskProps = {}, outsideClickDetectorProps = {}, onClose}) => (
-  <EuiOverlayMask headerZindexLocation="below" {...overlayMaskProps}>
-    <EuiOutsideClickDetector
-      onOutsideClick={onClose}
-      {...outsideClickDetectorProps}
-    >
-      <EuiFlyout
-        onClose={onClose}
-        {...flyoutProps}
-      >
-        {children}
-      </EuiFlyout>
-    </EuiOutsideClickDetector>
-  </EuiOverlayMask>
+export const WzFlyout = ({ children, flyoutProps = {}, onClose }) => (
```

Comment:
**question**: I saw you use `EuiFlyout` instead of this component. If this component is not used, maybe we could remove it or use it as a wrapper of custom `EuiFlyout`, for example, that has the `wzApp` that seems is included in all the flyouts.

Reply:
I think the better way is to remove and avoid maintaining the custom component `WzFlyout`. Also, Without the `Overlay` and the `OutsiderClickDetector`, the component doesn't add any behavior only works like the native `EuiFlyout`.